server/utils/validation/loginValidation.js

```diff
@@ -2,7 +2,7 @@ const yup = require('yup');
 
 const schema = yup.object().shape({
   email: yup.string().email().required(),
-  password: yup.string().min(4).required(),
+  password: yup.string().min(6).required(),
```

Comment:
I think we can enhance our password validation by adding a pattern like this 
```suggestion
  password: yup
    .string()
    .required('Please Enter your password')
    .matches(
      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
      "Must Contain 8 Characters, One Uppercase, One Lowercase, One Number and one special case Character"
    ),
```

Reply:
@Asmaathabet Great idea.  My additional suggestion is to remember to also show that message to the user in the frontend UI, so they know of the requirements *before* they fail validation :)